web/src/services/state/index.ts

```diff
@@ -75,6 +75,9 @@ export const useCurrentTheme = () => useAtom(currentTheme);
 const workspace = atom<Workspace | undefined>(undefined);
 export const useWorkspace = () => useAtom(workspace);
 
+const project = atom<Workspace | undefined>(undefined);
```

Comment:
Why project's type is `Workspce` and there's no type error?

Reply:
If we only need `projectId` we could just add a `projectId` atom instead of a `project`